doctr/models/utils/pytorch.py

```diff
@@ -18,9 +18,14 @@
     "export_model_to_onnx",
     "_copy_tensor",
     "_bf16_to_float32",
+    "_get_torch_compile_type",
 ]
 
 
+def _get_torch_compile_type() -> Any:
```

Comment:
@odulcy-mindee I'm not really happy with this change (especially because it's private - and can change over time) but we need to check the type.. any better option in mind ?

Reply:
@felixdittrich92 I don't understand what you try to achieve here. :thinking: The goal is to add `torch._dynamo.eval_frame.OptimizedModule` in the list of `allowed_archs` when we use the PyTorch Backend, so you wrap that in `_get_torch_compile_type`, isn't it ? What don't you like about this implementation?

Reply:
I don't see an alternative way here so asking ..maybe you have something more "elegant" in mind otherwise we keep it :sweat_smile:

Reply:
You can define this variable in `doctr/models/utils/pytorch.py`
```python
OptimizedModule = torch._dynamo.eval_frame.OptimizedModule
```
and import it in other module. It will be easier to maintain like that. You don't need to wrap it in a function.

So, a torch compiled model is instantiated as `OptimizedModule` ?

Reply:
correct :)